README.md

```diff
@@ -33,6 +33,7 @@ jobs:
   deploy_job:
     runs-on: ubuntu-latest
     name: Deploy app to Contentful
+    if: contains(github.ref, 'main') || contains(github.ref, 'master')
     steps:
       - uses: actions/checkout@v3
       - name: Use Node.js 16.x
```

Comment:
Should we change the to `Node.js 18.x` as we have changed in changelog.

Reply:
And we do a release then version will also change i.e. `v1.0.1`, may be we need to change that version in doc and change log also 🤔

Reply:
Ohh yes make sense, understood. we can make a release version tag to different tag.

Reply:
@vikaskumr the actual workflow is running 16 too, I'd suggest changing those in both Readme and the workflow, but that's beyond the scope of this ticket. I'll create an upgrade ticket for it